packages/loader/container-loader/src/container.ts

```diff
@@ -2453,6 +2446,34 @@ export class Container
 			this.runtime.setConnectionState(state && !readonly, this.clientId);
 		}
 	}
+
+	private handleDeltaConnectionArg(
+		connectionArgs: IConnectionArgs,
+		deltaConnectionArg?: "none" | "delayed",
```

Comment:
would it make sense to add these properties to IConnectionArgs

Reply:
I don't think so? The `IConnectionArgs` are for the `DeltaManager` and this `deltaConnectionArg` isn't really relevant to the DM, so it probably shouldn't be coupled to the DM.